File: py/fizz_buzz_may_2024.py
# fizz buzz solution May 2024

for i in range(1, 101):
    fizz = i % 3 == 0
    buzz = i % 5 == 0
    result = ""
    if fizz:
        result += "Fizz"
    if buzz:
        result += "Buzz"
    print(f"{result if result else i}")

File: c#/FizzBuzzMay2024.cs
// Fizz Buzz May 2024

public class FizzBuzzMay2024
{
	public static void fizzbuzz()
	{
		for (int i = 1; i <= 100; i++)
		{
			bool MultipleOf5 = i % 5 == 0;
			bool MultipleOf3 = i % 3 == 0;
			string output = "";
			if (MultipleOf3 && MultipleOf5) {
				output = "FizzBuzz";
			} else if (MultipleOf3) {
				output = "Fizz";
			} else if (MultipleOf5) {
				output = "Buzz";
			} else {
				output = i.ToString();
			}
			System.Console.WriteLine(output);
		}
	}

	public static void Main(string[] args)
	{
		fizzbuzz();
	}
}